yarn.lock

```diff
@@ -76,7 +76,7 @@
   version "22.2.3"
   resolved "https://registry.yarnpkg.com/@types/jest/-/jest-22.2.3.tgz#0157c0316dc3722c43a7b71de3fdf3acbccef10d"
 
-"@types/jss@^9.5.3":
+"@types/jss@^9.3.0", "@types/jss@^9.5.3":
```

Comment:
Are these changes needed?

Reply:
Yarn do it automatically. I only commit this changes